composer.json

```diff
@@ -27,8 +27,8 @@
     "php": ">=8.0",
     "league/flysystem": "^3.0",
     "monolog/monolog": "^2.9.1",
-    "php-di/php-di": "^7.0",
-    "symfony/console": "^v5.4.21"
+    "php-di/php-di": "^6.0",
+    "symfony/console": "^v5.0"
```

Comment:
it is needed for `/bin/wappspector.php` - the working example of script.

Reply:
it will work directly using the package, but the example is unnecessary when using it as a library. unfortunately, it is not possible to declare `bin-dev` files.